FF1Lib/OverworldMap.cs

```diff
@@ -367,6 +367,18 @@ public void ShuffleEntrancesAndFloors(MT19337 rng, IFloorShuffleFlags flags)
 					keepers.Add(OverworldTeleportIndex.Cardia5);
 				}
 
+				if ((bool)flags.NoFloater)

```

Comment:
If NoFloater is on without BahamutDock and Dragonshoard, there's still going to be orphaned chests. Could be a problem for shard hunt, or could lock players out of high level gear.
Perhaps it'd be better to enforce Dragonshoard and BahamutDock on floeaterless.

Reply:
I went with the more general solution, it now checks accessibility when placing loose items and shards, so if the Cardia islands are completely inaccessible, it won't place any incentivized items or shards there.